application/backend/src/main/resources/apps/bulma/components/buttons/buttons.html

```diff
@@ -16,13 +16,16 @@
   ~
 */-->
 
-<sly data-sly-use.model="pl.ds.bulma.components.models.ButtonsListComponent">
-  <div data-sly-unwrap="${!wcmmode.isEdit}" style="min-width: 125px; min-height: 50px">
+<sly data-sly-use.model="pl.ds.bulma.components.models.ButtonsComponent">
```

Comment:
There is a mechanism that adds a default placeholder for containers that have any HTML tag. In the case of some containers, it's important to add their styles to the empty placeholder, but in the case of buttons, I think we don't need that. So the component definition can be simplified to:
```
<sly data-sly-use.model="pl.ds.bulma.components.models.ButtonsComponent">
  <sly data-sly-test="${resource.hasChildren}">
    <div class="buttons ${model.buttonsClasses @ join=' '}">
      <sly
          data-sly-include="${'parsys.html' @ prependPath='/libs/wcm/foundation/components/parsys'}"></sly>
    </div>
  </sly>
</sly>```